src/coreclr/nativeaot/System.Private.CoreLib/src/System/InvokeUtils.cs

```diff
@@ -134,87 +134,301 @@ private static Exception ConvertOrWidenPrimitivesEnumsAndPointersIfPossible(obje
                 return CreateChangeTypeArgumentException(srcEEType, dstEEType);
             }
 
+            EETypeElementType srcElementType = srcEEType->ElementType;
+
+            // This switch supports the conversions present in the table below, in 'CanPrimitiveWiden'
             switch (dstElementType)
             {
+                // Boolean (W = BOOL)
```

Comment:
```suggestion
```
I do not think that these comments are useful

Reply:
Removed all of them in b804117848341a40e78a8ab1be7e63d385a39b63.